src/App.js

```diff
@@ -12,6 +12,7 @@ import {
 } from 'material-ui/Table';
 import { connect } from 'react-redux';
 import { isUrl, httpGet } from './fetch/fetchHelpers';
```

Comment:
Great! But usually, for common/helper functions, we name the folder, for example, `common`.